server/apps/immich/src/api-v1/asset/response-dto/asset.ts

```diff
@@ -0,0 +1,39 @@
+import { AssetEntity, AssetType } from '@app/database/entities/asset.entity';
+import { Exif, mapExif } from './exif';
+import { SmartInfo, mapSmartInfo } from './smart-info';
+
```

Comment:
Let's change the name of the files in this folder as well as the class name to explicitly telling the user what they are?

Reply:
👍